plugins/editor/src/editor/NativeHelpers.cpp

```diff
@@ -198,12 +198,16 @@ static std::vector<char *> createForkEnviron()
     return newEnv;
 }
 
-static constexpr char zenityPath[] = "/usr/bin/zenity";
+const char* zenityPath()
+{
+    static const char* zenityPath = g_find_program_in_path("zenity");
```

Comment:
Release glib's path with `g_free` after done.
other: zenityPath can be null if not found; fallback to the hardcoded value?

Reply:
Better like this?

Reply:
It's good